src/Compilers/CSharp/Portable/Syntax/TypeSyntax.cs

```diff
@@ -7,5 +7,7 @@ public abstract partial class TypeSyntax
         public bool IsVar => ((InternalSyntax.TypeSyntax)this.Green).IsVar;
 
         public bool IsUnmanaged => ((InternalSyntax.TypeSyntax)this.Green).IsUnmanaged;
+
+        public bool IsNotnull => ((InternalSyntax.TypeSyntax)this.Green).IsNotnull;
```

Comment:
>IsNotnull [](start = 20, length = 9)

Should this be "IsNotNull" (uppercase "N")? It looks like other members of the public API use uppercase for compound keywords ("IsReadOnly", "NameOf", "OrderBy", etc.).